app/assets/javascripts/datatables.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready( function () {$('#passengers').DataTable({
+  "aoColumns": [
+    null,
+    null,
+    null,
+    { "bSortable": false },
+    { "bSortable": false },
+    null,
+    { "bSortable": false },
+    { "bSortable": false },
+    { "bSortable": false },
+  ],
+  "aLengthMenu": [[25, 50, 75, 100, -1], [25, 50, 75, 100, "All"]],
+  "iDisplayLength": 100
+});});
```

Comment:
`aLengthMenu` determines what options are available for displaying X number of passengers in the table. `iDisplayLength` makes it default to whatever number you put there, in this case, 100. We did that because it resets to whatever the default is when you filter things ... setting to 100 makes it less visually jarring.